src/Microsoft.SqlTools.Credentials/CredentialsCommandOptions.cs

```diff
@@ -0,0 +1,36 @@
+﻿//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Globalization;
+using Microsoft.SqlTools.Hosting.Utility;
+
+namespace Microsoft.SqlTools.Credentials.Utility
+{
+    class CredentialsCommandOptions : CommandOptions
+    {
+        internal const string CredentialsServiceName = "MicrosoftSqlToolsCredentials.exe";
+
+        public CredentialsCommandOptions(string[] args) : base(args, CredentialsServiceName)
+        {
+        }
+
+        public override void SetLocale(string locale)
+        {
+            try
+            {
+                LocaleSetter(locale);
+
+                // Setting our internal SR culture to our global culture
+                SR.Culture = CultureInfo.CurrentCulture;
```

Comment:
This still seems a little odd to me - does the generated class not handle looking up the culture when fetching strings already?

Reply:
Apparently, the reason this is done is because in the tests, it attempts to access the Credential SR culture's name which is null of course if its not initialized. It does not change the localization however when it happens, I just tested. Therefore if we remove the culture name test, it should work without creating a new command class.